src/app/environment.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+from functools import lru_cache
+
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    host: str = "0.0.0.0"
+    port: str = "80"
+    app_reload: bool = True
+
+    # Jira
+    jira_base_url: str = "https://jira.allizom.org/"
+    jira_username: str
+    jira_password: str
+
+    # Bugzilla
+    bugzilla_base_url: str = "https://bugzilla-dev.allizom.org/"
+    bugzilla_api_key: str
+
+    # Logging
+    log_level: str = "info"
+
+
+@lru_cache()
+def get_settings() -> Settings:
+    return Settings()
+
+
+@lru_cache()
+def get_version():
+    """
+    Return contents of version.json.
+    This has generic data in repo, but gets the build details in CI.
+    """
+    _root = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+    version_path = os.path.join(_root, "version.json")
+    info = {}
+    if os.path.exists(version_path):
+        with open(version_path, "r", encoding="utf8") as version_file:
+            info = json.load(version_file)
```

Comment:
```suggestion
    info = {}
    version_path = Path(__file__).parents[2] / "version.json"
    if version_path.exists():
        info = json.loads(version_path.read_text(encoding="utf8"))
```

Don't know if I got that _quite_ right, but the general comment is that Pathlib might be nice here

Reply:
Made an issue for this; I'd like to look back at this when we've got the build running and deploying.